uv.lock

```diff
@@ -1,5 +1,5 @@
 version = 1
-revision = 3
+revision = 2
```

Comment:
not really blocking, but it'd be nice if we don't constantly change the revision

Reply:
I just followed the readme instructions:
https://github.com/randovania/randovania/blob/main/README.md?plain=1#L235-L238

Is there a command to update a dep and leave the revision alone, or are ya'll just manually editing the file every time?

Reply:
By keeping uv up to date. Should be `uv self update` iirc